Jenkinsfile

```diff
@@ -48,7 +48,7 @@ pipeline {
         /* last version before the 1.14 update in 28630 */
         CLEAR_LINUX_VERSION_1_13 = "28620"
 
-        PMEM_PATH = "/go/src/github.com/intel/pmem-csi"
```

Comment:
The error message says "go mod tidy; go mod vendor". Isn't that the real problem, not where the source is located? I'm not sure (doing it makes some changes to go.sum, but that may be due to slightly different Go versions - I am using 1.13.4).

The code should still build inside GOPATH. Moving it elsewhere smells like a workaround that doesn't address the real issue.

Reply:
It was not GOPATH. GOROOT is the real cause:  https://github.com/golang/go/issues/34657

Reply:
So the problem was that the Jenkinsfile and the Dockerfile both used /go, one for the source code (aka GOPATH), the other for the Go compiler (and thus GOROOT)? I wonder where GOPATH and/or HOME are set. It would be nice to avoid that overlap.

In the meantime, choosing some other directory is fine. However, can you add a comment explaining that PMEM_PATH is used as location for the source to build outside of GOPATH?

Reply:
> So the problem was that the Jenkinsfile and the Dockerfile both used /go, one for the source code (aka GOPATH), the other for the Go compiler (and thus GOROOT)? I wonder where GOPATH and/or HOME are set. It would be nice to avoid that overlap.

As of my understanding with go modules, GOPATH is irrelevant. That means there is no more hart need of keeping source code inside GOPATH. The only problem with GOROOṬ. In our both(Jenkinsfile & Dockerfile) cases GOROOT is pointed(I guess) to /go.